libV2/schemaUtils.js

```diff
@@ -576,6 +576,11 @@ let QUERYPARAM = 'query',
           return;
         }
 
+        // Set type to binary
+        if (property.format === 'binary') {
+          property.type = 'binary';
```

Comment:
@thim81 We should not set `type` to an invalid JSON schema type. This can create issues with other modules that we use.

Instead, we can keep the format `binary` that's being removed and while determining text/file field make sure to add file when type=string and format=binary. 

There are few changes needed around codebase for this but I can make them. Could you provide me access to your fork for making these changes in this PR?

Reply:
Hi @VShingala You should have write-access:
![IMG-2024-07-18 at 17-25-07](https://github.com/user-attachments/assets/c4950661-855b-4d1a-9a8f-e38109d618dc)